### PR TITLE
General Bugfix for Tag/Untag

### PR DESCRIPTION
**Pull request by krusagiz**
- Added check for multiple index given for Tag/Untag command
- Added new `getSingleValue` utility method in **ArgumentMultimap**
to get a singular value that is checked for extras
- Added new message format in **Messages**

### CODE REVIEW

**Comment by krusagiz** [comment]
SLAP'ed the Tag/Untag Parsers

File: src/main/java/thrift/logic/parser/UntagCommandParser.java [hunk 1]
```diff
@@ -28,21 +27,20 @@ public UntagCommand parse(String args) throws ParseException {
...
+            String indexStr = argMultimap.getSingleValue(CliSyntax.PREFIX_INDEX)
+                    .orElseThrow(() -> new ParseException(""));
+            index = ParserUtil.parseIndex(indexStr);
+            tagSet = ParserUtil.parseTags(argMultimap.getAllValues(CliSyntax.PREFIX_TAG));
+            if (tagSet.isEmpty()) {
+                throw new ParseException(UntagCommand.MESSAGE_NOT_UNTAGGED);
             }
         } catch (ParseException pe) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, UntagCommand.MESSAGE_USAGE), pe);
-        } catch (IllegalArgumentException iae) {
```
Comment (lye-jw) [inline src/main/java/thrift/logic/parser/UntagCommandParser.java:43]: Is it alright to remove the catch block for `IllegalArgumentException` here? If it never catches anything in your latest version disregard this comment.

**Review by lye-jw — APPROVED** [review]
LGTM!

File: src/main/java/thrift/logic/parser/UntagCommandParser.java (hunk 1 above)
Comment (krusagiz) [reply src/main/java/thrift/logic/parser/UntagCommandParser.java:43]: The block (for-loop for tagSet) that throws the `IllegalArgumentException` has been removed from the code so it doesn't throw it anymore! 👍

**Comment by krusagiz** [comment]
@lye-jw thank you for the merge!

**Pull request merged**